src/lib/CustomRulePathManager.ts

```diff
@@ -36,19 +42,20 @@ export class CustomRulePathManager implements RulePathManager {
 
 		this.logger.trace(`Initializing CustomRulePathManager.`);
 		// Read from the JSON and use it to populate the map.
-		let data = null;
+		let data: string = null;
 		try {
 			data = await this.readRulePathFile();
 		} catch (e) {
+			const err: NodeJS.ErrnoException = e as NodeJS.ErrnoException;
```

Comment:
The top-level `Error` type doesn't have a `code` property. It's extended by this type that lives in the `NodeJS` namespace, and that type has a code. So we cast.